.github/workflows/test.yml

```diff
@@ -32,8 +32,14 @@ jobs:
           yarn test:playwright
           yarn test:cypress
 
-      - name: Publish to Chromatic
+      - name: Publish to Chromatic (Playwright)
         uses: chromaui/action@v1
         with:
           projectToken: ${{ secrets.CHROMATIC_PLAYWRIGHT_PROJECT_TOKEN }}
-          buildScriptName: build-archive-storybook
+          buildScriptName: build-archive-storybook:playwright
+
+      - name: Publish to Chromatic (Cypress)
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_CYPRESS_PROJECT_TOKEN }}
+          buildScriptName: build-archive-storybook:cypress
```

Comment:
Maybe not worth doing here, but should we move these to separate workflow files so we can run them in parallel?

Reply:
That's a fair point. Would you suggest a separate file for Playwright and Cypress (and in that case, one for unit tests as well)?

I had a stuck build on Playwright and it prevented Cypress results from even showing up, so I could see that being useful. I guess it would require us to install dependencies more times. WDYT?

Reply:
Yeah, I think we could have each as a separate workflow, and have the playwright and cypress workflows depend on the unit test one so that we don't bother with chromatic builds if the unit tests are failing.